### PR TITLE
Update CONTRIBUTING.md with new location of dashboard and workspace loader

### PR DESCRIPTION
**Pull request by skabashnyuk**
### What does this PR do?
Update CONTRIBUTING.md with new location of dashboard and workspace loader

### What issues does this PR fix or reference?
n/a

#### Release Notes
n/a

#### Docs PR
n/a

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -148,10 +148,10 @@ In this section we are going to start a new workspace to work on workspace loade
...
 
 ```bash
-$ chectl workspace:start --devfile=https://raw.githubusercontent.com/eclipse/che/master/workspace-loader/devfile.yaml
+$ chectl workspace:start --devfile=https://raw.githubusercontent.com/eclipse/che-workspace-loader/master/devfile.yaml
 ```
 
-or as a factory `https://<CheInstance>/f?url=https://raw.githubusercontent.com/eclipse/che/master/workspace-loader/devfile.yaml`
+or as a factory `https://<CheInstance>/f?url=https://raw.githubusercontent.com/eclipse/che-workspace-loader/master/devfile.yaml
 
 At workspace start, Che will clone Che source code (including the workspace-loader)
```
Comment (sparkoo) [inline CONTRIBUTING.md]: No longer true. And the devfile is broken now as it has Che project with `workspace-loader` as `sparseCheckoutDir`, which does not exist now.

**Comment by sparkoo** [comment]
Thanks for this. It's a big relief not to wait for sloooooow dashboard build. :heart:

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -56,12 +56,12 @@ Let's cover the developer flow for theses projects:
 
 ### Dashboard
```
Comment (sunix) [inline CONTRIBUTING.md:57]: Maybe this part has to move the a dedicated CONTRIBUTING.md in the dashboard repo. WDYT ?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -129,12 +129,12 @@ $ node_modules/.bin/gulp serve --server=<che_api_url>
 
 ### Workspace Loader
 
-Workspace loader source code is located in [<this.repo>/workspace-loader](./workspace-loader/) folder.
+Workspace loader source code is located in [https://github.com/eclipse/che-workspace-loader](https://github.com/eclipse/che-workspace-loader) folder.
 Here is the developer workflow if you want to contribute to it:
 
 #### Devfile for workspace loader development
```
Comment (sunix) [inline CONTRIBUTING.md:133]: Maybe this part has to move the a dedicated CONTRIBUTING.md in the workspace loader repo. WDYT ?

**Review by sunix — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (skabashnyuk) [reply CONTRIBUTING.md:57]: separate issue? CC @sleshchenko

File: CONTRIBUTING.md (hunk 3 above)
Comment (skabashnyuk) [reply CONTRIBUTING.md:133]: separate issue? CC @sleshchenko

**Review by akurinnoy — APPROVED** [review]
my +1 to merge it as is, and then open separate issues to move corresponding parts of `CONTRIBUTING.md` to `che-dashboard` and `che-workspace-loader` repos.

**Review by sparkoo — APPROVED** [review]

File: CONTRIBUTING.md (hunk 3 above)
Comment (sunix) [reply CONTRIBUTING.md:133]: https://github.com/eclipse/che/issues/16432

File: CONTRIBUTING.md (hunk 2 above)
Comment (sunix) [reply CONTRIBUTING.md:57]: https://github.com/eclipse/che/issues/16431

**Review by sunix — APPROVED** [review]

**Comment by sunix** [comment]
created https://github.com/eclipse/che/issues/16431 and https://github.com/eclipse/che/issues/16432

**Pull request merged**